chart/templates/controller-manager.yaml

```diff
@@ -202,7 +202,8 @@ spec:
         - --leader-elect
         - --wds-name={{.Values.ControlPlaneName}}
         - --api-groups={{.Values.APIGroups}}
-        image: ghcr.io/kubestellar/kubestellar/controller-manager:0.20.0-alpha.1
+        - -v=2
+        image: ghcr.io/kubestellar/kubestellar/controller-manager:0.20.0-rc1
```

Comment:
The tag here is just a placeholder (replaced by the release workflow and the local test automation), right? If so then it should explicitly say that it is a placeholder. Perhaps a tag like `will-be-replaced`.

Reply:
ok, done